docs/README.md

```diff
@@ -0,0 +1,72 @@
+# Package submission guidelines
+
+To submit a package, simply make a pull request with the package to this
+repository. Start by cloning the repository (we recommend using a [sparse
+checkout][sparse-checkout]), and create a new directory for your package named
+`packages/{namespace}/{package-name}/{version}`. For now, the only allowed
+namespace is `preview`. You can then [copy your package files][copy] here,
+commit your changes and open a pull request.
+
+We have guidelines that you have to follow to be accepted in Typst Universe.
+They ensure that the published package meet some quality standards and work
+properly for everyone. Please note that this list may be extended over time as
+improvements/issues to the process are discovered. Given a good reason, we
+reserve the right to reject any package submission.
+
+The following links will teach you what to pay attention to when working on:
+
+- your [package manifest][manifest]
+- your [Typst files][typst] (including template and example files)
+- [images, fonts and other assets][resources]
+- the [README file, and documentation in general][documentation]
+- the [license] of your package
+
+On top of that, there are some general requirements for getting a package
+published:
+
+- **Functionality:** Packages should conceivably be useful to other users and
+  should expose their capabilities in a reasonable fashion.
+
+- **Security:** Packages must not attempt to exploit the compiler or packaging
+  implementation, in particular not to exfiltrate user data.
+
+- **Safety:** Names and package contents must be safe for work.
+
+- **Size:** Packages should not contain large files or a large number of files.
+  This will be judged on a case-by-case basis, but if it needs more than ten
+  files, it should be well-motivated. To keep the package small and fast to
+  download, please `exclude` images for the README or PDF files with
+  documentation from the bundle. Alternatively, you can link to images hosted on
+  a githubusercontent.com URL (just drag the image into an issue).
```

Comment:
I feel like this a bit outdated, quite a lot of packages have more than ten files now. I also wrote more precise rules on what should be included in the repository or excluded from the archive.

Reply:
Agreed. We can keep it more generic. It should state somewhere that we won't accept packages with a large number of files, but the exact amount is decided on a case-by-case basis.